### PR TITLE
Fix cupy version check

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

File: version.py [hunk 1]
```diff
@@ -27,4 +29,6 @@ def get_chainer_version():
 
 def get_cupy_version():
     setup_path = os.path.join(os.path.dirname(__file__), 'cupy', 'setup.py')
+    if setup_path is None:
```
Comment (unnonouno) [inline version.py]: `setup_path` is not `None`. Check result of `get_version`

File: docker.py [hunk 2]
```diff
@@ -41,7 +41,8 @@ def get_cuda_cudnn_choices(target, with_dummy=False):
         choices = [('none', 'none')] + choices
 
     cupy_major = version.get_cupy_version()[0]
-    if cupy_major >= 2:
+
+    if cupy_major is not None and cupy_major >= 2:
```
Comment (unnonouno) [inline docker.py]: When `get_cupy_version` returns `None`, you need to check its result

**Comment by okuta** [comment]
I fixed

File: version.py [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
 
 def get_version(setup_path):
+    if not os.isfile(setup_path):
```
Comment (unnonouno) [inline version.py]: `os.path.isfile`

Comment (okuta) [reply version.py]: I'm sorry, I made a easy mistake. I fixed

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**